tests/data-access/test_wdqs_reader.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from SPARQLWrapper import SPARQLWrapper
+
+from wikidatarefisland.data_access import WdqsReader
+
+
+class MockResponse:
+    @staticmethod
+    def convert():
+        return {"results": {"bindings": [{'a': 'b'}]}}
+
+
+@pytest.fixture
+def mock_response(monkeypatch):
```

Comment:
I love this, it's the coding equivalent of doing a death drop in a voguing ball.

![](https://media.giphy.com/media/eJ4jA6TKADyQsgBggc/giphy.gif)

Reply:
I'm going to miss these comments in gerrit :((